build/index.html

```diff
@@ -16,7 +16,7 @@
 
     <!-- Button component -->
 
-    <button type="button" class="btn btn-primary">Primary</button>
+    <button type="button" class="btn dsf-btn-primary">Primary</button>
```

Comment:
These buttons shouldn't need to inherit bootstrap `.btn` class as we inherit that into `dsf-btn-*`

Reply:
done, we extends bootstrap .btn class in dsf-btn-primary